rental.rb

```diff
@@ -0,0 +1,22 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+    book.rentals_list.push(self)
+    person.rentals_list.push(self)
+  end
+
+  def add_person(person)
+    @person = person
+    person.rentals_list.push(self) unless person.rentals_list.include?(self)
+  end
+
+  def add_book(book)
+    @book = book
+    book.rentals_list.push(self) unless book.rentals_list.include?(self)
+  end
```

Comment:
- [ ] There is no need for the add_book method in the Rental class since you defined the instance variable @book and pushed the rental itself to the book rentals_list in the initialize method:
```rb
  def initialize(date, person, book)
    @date = date
    @person = person 
    @book = book # here
    book.rentals_list.push(self)   # and here
    person.rentals_list.push(self)
  end
```